anago

```diff
@@ -623,7 +618,7 @@ local_kube_cross () {
   local local_image
   local docker_image
 
-  docker_image="${GCRIO_PATH_PROD}/kube-cross"
+  docker_image="${GCRIO_PATH_TEST}/kube-cross"
```

Comment:
I don't think we want to change that. As far as I can tell the intention here is to get the canonical kubecross image for whatever version of `k/k` we've checked out and just tag that locally for easier usage. We don't push that tag. I think we do want to use the prod image, and not any test image; we want to use [this] image in all cases.

[this]:  https://github.com/kubernetes/kubernetes/tree/master/build/build-image/cross